database/Seeders/eggs/source-engine/egg-insurgency.json

```diff
@@ -1,24 +1,29 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1"
+        "version": "PTDL_v1",
+        "update_url": null
     },
-    "exported_at": "2020-10-19T23:35:42+00:00",
+    "exported_at": "2021-06-05T16:38:31-04:00",
     "name": "Insurgency",
     "author": "support@pterodactyl.io",
     "description": "Take to the streets for intense close quarters combat, where a team's survival depends upon securing crucial strongholds and destroying enemy supply in this multiplayer and cooperative Source Engine based experience.",
-    "image": "quay.io\/pterodactyl\/core:source",
-    "startup": ".\/srcds_run -game {{SRCDS_GAME}} -console -port {{SERVER_PORT}} +map {{SRCDS_MAP}} +ip 0.0.0.0 -strictportbind -norestart",
+    "features": null,
+    "images": [
+        "ghcr.io\/pterodactyl\/games:source"
+    ],
+    "file_denylist": [],
+    "startup": ".\/srcds_run -game insurgency -console -port {{SERVER_PORT}} +map {{SRCDS_MAP}} +ip 0.0.0.0 -strictportbind -norestart",
     "config": {
         "files": "{}",
-        "startup": "{\"done\": \"gameserver Steam ID\", \"userInteraction\": []}",
-        "logs": "{\"custom\": true, \"location\": \"logs\/latest.log\"}",
+        "startup": "{\r\n    \"done\": \"gameserver Steam ID\",\r\n    \"userInteraction\": []\r\n}",
```

Comment:
userInteraction doesn't exist anymore, it is replaced by the `features` of the egg. Could remove it since you are updating all eggs at once

Reply:
Removed userInteraction, was still present in the egg shipped with the panel.